src/main/game/gameboard/areaEffect/AreaEffectManager.java

```diff
@@ -0,0 +1,31 @@
+package game.gameboard.areaEffect;
+
+import java.util.ArrayList;
+
+/**
+ * Created by khariollivierre on 3/13/17.
+ */
+
+/*
+TODO: Specify number of effects to place
+TODO: Handle effect placement
+TODO: Handle effect actions
+ */
+
+public class AreaEffectManager {
+    private ArrayList<AreaEffect> poison;
```

Comment:
what's this for? I don't think we'll need to grab all area effects for anything. I think we just need a way to place them on a tile.